test/over_react_redux/connect_flux_test.dart

```diff
@@ -691,7 +722,53 @@ main() {
         },
       };
 
-      testParameterCases(testCases);
```

Comment:
Inlined this so it could access closure variables; it was only being called on this line, anyways.